content/_help/verify-your-identity/accepted-state-issued-identification._es.md

```diff
@@ -1,32 +1,27 @@
 ---
-title: Documentos de identidad estatales aceptados
+title: Documentos de identificación aceptados
 category: verify-your-identity
 permalink: /es/help/verify-your-identity/accepted-state-issued-identification/
 order: 2
+do_list:
+  - "**Licencia de conducir** válida en los 50 estados, el Distrito de Columbia (DC) y otros territorios de Estados Unidos (Guam, Islas Vírgenes de Estados Unidos, Samoa Americana, Islas Marianas y Puerto Rico)"
+  - "**Una tarjeta de identificación estatal para no conductores.** Este es un documento de identidad emitido por el estado, el Distrito de Columbia (DC) o el territorio de EE. UU. que hace valer la identidad pero no otorga privilegios de conducción."
+alert: <strong>Si no cuenta con una licencia de conducir válida o con un documento de identidad estatal, no puede utilizar Login.gov para verificar su identidad.</strong> Por favor, comuníquese con el centro de ayuda de la entidad asociada para saber qué puede hacer como alternativa.
+dont_list:
+  - No podrá pasar la verificación de identidad si su documento de identidad está caducado.
+  - Los documentos de prórroga no pueden utilizarse en lugar de un documento de identidad vigente y no caducado.
+  - No, no puede usar un documento de identidad temporal o de papel.
```

Comment:
Could you also add a `&nbsp;` before "papel" so that word can stay together with the first line?